src/v/coproc/event_listener.cc

```diff
@@ -103,6 +130,18 @@ decompress_wasm_events(model::record_batch_reader::data_t events) {
 }
 
 ss::future<> event_listener::do_start() {
+    bool connected = co_await _client.is_connected();
+    if (!connected) {
+        try {
+            co_await _client.connect();
+        } catch (const kafka::client::broker_error& e) {
+            throw e;
+        }
```

Comment:
Were you planning to do something here, or perhaps debug code that wasn't cleaned up?